src/python/pants/backend/python/target_types.py

```diff
@@ -393,6 +393,8 @@ class PexEnvField(DictStringToStringField):
 
 class PexPlatformsField(StringSequenceField):
     alias = "platforms"
+    removal_version = "2.20.0.dev0",
```

Comment:
Aha, it's the commas. You made 1-tuples on this line and the next.

Reply:
yep, my brain isn't working so well right now

Reply:
But also it turns out something may also be broken in the deprecation logic. As well as my brain.

Reply:
Yep: [yeeeesh](https://github.com/pantsbuild/pants/blob/ee6f123d05a0a79adff04aaadecaa43ff761f1e3/src/rust/engine/src/externs/target.rs#L205)

Will fix.

Reply:
https://github.com/pantsbuild/pants/pull/20014